src/data-access/db.ts

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+import { Sequelize, Dialect } from 'sequelize';
+
+dotenv.config();
```

Comment:
```suggestion
import { Sequelize, Dialect } from 'sequelize';
```

It's enough to init dotenv once in the entry file which is the app.ts